exercises/wordy/src/test/java/WordProblemSolverTest.java

```diff
@@ -112,12 +118,65 @@ public void testUnknownOperation() {
 
     @Ignore("Remove to run test")
     @Test
-    public void testInvalidQuestionFormat() {
+    public void testNomMathQuestion() {
         expectedException.expect(IllegalArgumentException.class);
```

Comment:
There is a small typo, it should be "testNonMathQuestion".

Reply:
Sorry, I'm going to fix it now.